pkgs/build-support/rust/import-cargo-lock.nix

```diff
@@ -63,7 +63,13 @@ let
 
   # We can't use the existing fetchCrate function, since it uses a
   # recursive hash of the unpacked crate.
-  fetchCrate = pkg: fetchurl {
+  fetchCrate = pkg:
+    assert lib.assertMsg (pkg ? checksum)
+      ''
+      package ${pkg.name} does not have a checksum.
+      Please note that the Cargo.lock format where checksums used to be listed under [metadata] is not supported.
+      If that is the case, running `cargo update` with a recent toolchain will automatically update the format along with the crate's depenendencies.'';
+    fetchurl {
```

Comment:
```suggestion
  fetchurl {
```

Reply:
I indented the attrs in the args passed to fetchurl instead of deindenting fetchurl, to keep fetchurl one level deeper than fetchCrate